lib/broccoli/translation-reducer.js

```diff
@@ -114,6 +115,7 @@ class TranslationReducer extends CachingWriter {
       }
 
       let translation = readAsObject(translationPath);
+      translation = stripNestedNulls(translation);
```

Comment:
I'm not sure if this is best done here or outside the `readAsObject` method.

Also this could be refactored to `stripNestedNulls(readAsObject(translationPath))` (need pipeline operator :cry: )